### ISSUE TITLE
Implement Collections

### ISSUE THREAD

**Issue by TheVillageGuy** [MbinOrg/mbin#547]
There are over 5 magazines/communities I'm aware of so far for cities skylines, to give you an example. Combining them somehow using activityPub is miles away and requires all software to cooperate. Instead, mbin displays them in a combined magazine, not changing anything in the existing fediverse system.

**On front page**
-entries are displayed only once, linking to the entry in the combined magazine
-upvotes (& downvotes) are sent to all subscribed magazines
-upvote (& downvote) counts are accumulated

**On combined magazines**
-all original magazine entries are displayed on one combined magazine, only once
-every entry has a coloured outline, so it's easy to tell from which magazine it originates, and possibly other indicators
-every magazine part of this combination can be subscribed/unsubscribed from easily with sliders in the sidebar
-not being subscribed to one means it will not show entries in the combined magazine
-upvotes (& downvotes) are sent to all subscribed magazines
-upvote (& downvote) counts are accumulated
-activity cannot be accessed (maybe in the future?)

**Outside the combined magazine**
-searching for the relevant magazines only leads to the combined one, the original magazines will not appear anywhere. They can however be accessed from the combined magazine by clicking on their names next to the slider

**viewing an original magazine**
-can be done by clicking the name next to the sliders in the combined magazine, including unsubscribed magazines
-It is clearly indicated the magazine is part of a collection and it's easy to jump to the combined magazine.
-Commenting is possible
-deleting / editing is not - or just opens the editor for the combined magazine
-Up/down votes are sent to all relevant magazines (not sure if there's any use for this)
-activity can be accessed

**Creating an entry**
-entries are kept sync in all combined magazines
-entries can be made in a selection of the magazines, using sliders before posting the entry

**Editing**
-Editing is not possible from the original magazines, only from the combined view, to prevent de-syncing
-While editing an entry it is possible to change the selection, which will, depending on the situation per magazine, delete, create or edit the post on all relevant magazines when clicking done. Deselecting all magazines is not possible, as this would technically be a complete delete action. 
-entriess can not be edited from their individual magazines

OR

-editing is only applied to existing entriess in all relevant magazines, disregarding current subscription status

**Deleting**
-deleting an entry deletes it from all available magazines, regardless of current subscription, to ensure full deletion
-entries cannot be deleted from their individual magazines

**Commenting**
-Parent comments can be placed in all existing entriess from the combined magazine
-replies can only be made to the magazine where the comment originates from, as it's impossible to duplicate comments in such a way that the original author will still be able to edit it. (Unless it's a local user, which includes the author?) When replying it is shown in what magazine the reply is being made in.
-Commenting can be done when viewing the original magazines

Impressions:

![image](https://github.com/MbinOrg/mbin/assets/47496248/e53fb3d7-205f-4439-9c1e-e1db67be300d)
![image](https://github.com/MbinOrg/mbin/assets/47496248/6da12dc9-2f88-43be-9dcd-3c5414d376f5)

**Comment by melroy89** [comment]
Duplicate of: https://github.com/MbinOrg/mbin/issues/486 ?

**Comment by aaronliu0130** [comment]
._. seems like it

**Comment by melroy89** [comment]
CLose as dup
